com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -129,6 +130,21 @@ def _map(self, func, schema):
         oport = op.addOutputPort(schema=schema)
         return Stream(self.topology, oport)
 
+    def view(self, buffer_time = 10.0, sample_size = 10000):
+        """
+        Defines a view on a stream. Returns a view object which can be graphed after the jobs is submitted.
```

Comment:
Need to expand this more, what is a view. I don't really think the view object can be graphed, it provides access to the data exposed by the view, which then can be graphed. I.e that comment is very specific to one environment, Jupyter.